assets/xml/objects/object_mo.xml

```diff
@@ -50,6 +50,7 @@
         <DList Name="gMorphaTentaclePart38DL" Offset="0x87A8"/>
         <DList Name="gMorphaTentaclePart39DL" Offset="0x87F0"/>
         <DList Name="gMorphaTentaclePart40DL" Offset="0x8838"/>
+        <Texture Name="gMorphaAndPoolWaterTex" Format="rgba16" Width="32" Height="32" Offset="0x8870"/>
 
```

Comment:
we can prob just say gMorphaWaterTex, and it will be implied that it is used for both the pool and morpha itself because they both belong to the same body of water

Reply:
I think:
`gMorphaBaseTentacleAndPoolWaterTex`
would be more suitable.

Reply:
well, my point is that it probably doesnt need to be more specific cause its just a water texture. and morpha is made of water, so that is already a given

Reply:
Are you sure it wouldn't be better to be like: "Hey, this texture is used twice" in the name?

Reply:
Or perhaps a comment?

Reply:
I dont think its needed personally, but we can see what other people think

Reply:
It would be nice to indicate that it is used for both somewhere. Either a comment or the name is fine to me.

Reply:
comment is probably preferred. Textures get reused pretty frequently, and it would be impossible to indicate all of the uses in its name in some cases. 
imo a comment is also not really needed cause if you want to see occurences you can just search for it. but i would compromise for a comment over the name

Reply:
The only reason I would like to have the comment if we dont make the name specific is that if you global search for it VSCode (and I believe VS2022) wont look at the actual DLists by default but it will look at the XML.

Reply:
if youre searching for anything in an xml its a given that you need to turn off the gitignore exclusion

Reply:
its fair to not know that the some text editors work this way, of course. but you would know something is up when searching and not getting hits, and probably ask someone about it. This happened to mega the other day when i explained the feature to him.
It shouldnt be a justification for a naming scheme, is more my point

Reply:
Yea, I've never noticed this git ignore issue. It was certainly breaking my brain.
It's my understanding that this has never been an issue for sm64 nor mk64.

Reply:
my understanding is that dlists are not extracted and are included in the repo right?
the whole assets folder is gitignored for now, though the plan is to commit the files themselves and only ignore the files that contain the extracted assets that are deemed too sensitive

Reply:
Also, keep in-mind. That technically the *second* use of this texture is not the same texture. It's rendered as an i8. So it's different. It's not the same.

Either way. I don't really care. Would it be possible to put a rush on this PR? Whatever name and comment or not is fine with me.